packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -764,6 +778,8 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 
     private processInboundMessage(message: ISequencedDocumentMessage): void {
         const startTime = Date.now();
+        assert(!this.opsCurrentlyProcessing, "Currently processing ops.");
```

Comment:
nit: I'd say something like "Already" not "Currently", or otherwise explain that the assert means something about reentrancy on this function.  Doesn't really matter because it'll turn into an assert code though.